TF_Tooling_Web/src/InputControllers/WebInputController.ts

```diff
@@ -119,14 +119,21 @@ export class WebInputController extends BaseInputController {
         switch (_inputData.InputType) {
             case InputType.CANCEL:
                 this.ResetScrollData();
-                let cancelEvent: PointerEvent = new PointerEvent("cancel", this.activeEventProps);
+                let cancelEvent: PointerEvent = new PointerEvent("pointercancel", this.activeEventProps);
+                let outEvent: PointerEvent = new PointerEvent("pointerout", this.activeEventProps);
+
+                if (this.lastHoveredElement !== null) {
```

Comment:
I thinks this should also check that it isn't elementAtPos as well as otherwise we may get duplicate messages